### ISSUE TITLE
the codes of matcher/alias are useless

### ISSUE THREAD

**Issue by masonExcelsia** [excelsia/vee#133]
I found that the codes of matcher/alias are useless and when the project is built or tested, the two part of codes will also be contained. I suppose these useless code will affect the efficient and ability of our system. Could we delete these code from our project? @ncying @zhangdong0620 @zlvee

**Comment by ncying** [comment]
yes. It is now unused in our system. but just keep for easy extension and some further works. this can be discussed later.
